pkg/util/updater/updater.go

```diff
@@ -207,321 +185,53 @@ func (u *Updater) getVersion(updateConfig UpdateConfig) (string, error) {
+		u.log.Error("Get error during remove skywire-bin package")
+		return err
 	}
-
-	return checksums[first:last], nil
+	return nil
 }
 
-func (u *Updater) downloadChecksums(url string) (checksums string, err error) {
-	resp, err := http.Get(url) // nolint:gosec
-	if err != nil {
-		return "", err
-	}
-
-	defer func() {
-		if closeErr := resp.Body.Close(); closeErr != nil && err == nil {
-			err = closeErr
-		}
-	}()
-
-	if resp.StatusCode != http.StatusOK {
-		return "", fmt.Errorf("received bad status code: %d", resp.StatusCode)
-	}
-
-	capturer := stdio.NewCapturer()
-	stdoutWriter, err := capturer.CaptureStdout()
-	if err != nil {
-		return "", err
-	}
-
-	defer func() {
-		if err := capturer.Release(); err != nil {
-			u.log.Errorf("Failed to release output: %w", err)
-		}
-	}()
-
-	r := io.TeeReader(resp.Body, u.progressBar(stdoutWriter, resp.ContentLength, checksumsFilename))
-
-	data, err := ioutil.ReadAll(r)
-	if err != nil {
-		return "", err
+func (u *Updater) aptInstall() error {
+	if err := exec.Command("sudo", "apt", "install", "skywire-bin", "-y").Run(); err != nil {
+		u.log.Error("Get error during installing skywire-bin package")
+		return err
 	}
-
-	return string(data), nil
-}
-
-func (u *Updater) progressBar(w io.Writer, contentLength int64, filename string) io.Writer {
-	width := progressbar.OptionSetWidth(0)
-	desc := progressbar.OptionSetDescription("Downloading " + filename)
-	speed := progressbar.OptionSetBytes64(contentLength)
-	theme := progressbar.OptionSetTheme(progressbar.Theme{})
-	writer := progressbar.OptionSetWriter(io.MultiWriter(w, u.status))
-	completion := progressbar.OptionOnCompletion(func() { fmt.Printf("\n") })
-
-	return progressbar.NewOptions64(contentLength, speed, completion, width, theme, desc, writer)
+	return nil
 }
 
-func (u *Updater) downloadFile(url, filename string) (path string, err error) {
-	resp, err := http.Get(url) // nolint:gosec
-	if err != nil {
-		return "", err
-	}
-
-	defer func() {
-		if closeErr := resp.Body.Close(); closeErr != nil && err == nil {
-			err = closeErr
-		}
-	}()
-
-	if resp.StatusCode != http.StatusOK {
-		return "", fmt.Errorf("bad HTTP response status code %d", resp.StatusCode)
-	}
-
-	tmpDir := os.TempDir()
-	path = filepath.Join(tmpDir, filename)
-
-	if _, err = os.Stat(path); err == nil {
-		// File exists
-		if err := os.Remove(path); err != nil {
-			return "", err
-		}
-	}
-
-	f, err := os.Create(path)
-	if err != nil {
-		return "", err
-	}
-
-	capturer := stdio.NewCapturer()
-	stdoutWriter, err := capturer.CaptureStdout()
-	if err != nil {
-		return "", err
-	}
-
-	defer func() {
-		if err := capturer.Release(); err != nil {
-			u.log.Errorf("Failed to release output: %w", err)
-		}
-	}()
-
-	out := io.MultiWriter(f, u.progressBar(stdoutWriter, resp.ContentLength, filename))
-
-	if _, err := io.Copy(out, resp.Body); err != nil {
-		return "", err
+func (u *Updater) restartBoard() {
```

Comment:
Is a restart needed?

Reply:
Yes, I thought, based on logic of Moses Debian package. I'll check it with him again.

Reply:
I change it to restart service, not the board at all.